tests/meta/demultiplex/test_validation.py

```diff
@@ -130,45 +134,42 @@ def test_validate_flow_cell_delivery_status_forced(tmp_path: Path):
     )
 
 
-def test_validate_samples_have_fastq_files_passes(mocker, novaseqx_demultiplexed_flow_cell: Path):
-    """Test the check of a flow cells with fastq files does not raise an error."""
-    # GIVEN a demultiplexed flow cell with fastq files
-    flow_cell_with_fastq = FlowCellDirectoryData(flow_cell_path=novaseqx_demultiplexed_flow_cell)
+def test_validate_samples_have_fastq_files_passes(
```

Comment:
This test was changed as it used a flow cell with all the samples having fastq files. Changed to only one sample having one fastq file.